roles/splunk_common/tasks/start_splunk.yml

```diff
@@ -1,20 +1,24 @@
 ---
-- name: Start Splunk via cli
+- name: "Start Splunk via cli"
   command: "{{ splunk.exec }} start --accept-license --answer-yes --no-prompt"
   become: yes
   become_user: "{{ splunk.user }}"
   when: not splunk.enable_service
 
-- name: Start Splunk via service
+- name: "Start Splunk via service"
   service:
     name: splunk
     state: restarted
   when: splunk.enable_service and ansible_system is match("Linux")
   become: yes
   become_user: root
 
-- name: Start Splunk via Windows service
+- name: "Start Splunk via Windows service"
   win_service:
     name: splunkd
     state: restarted
   when: splunk.enable_service and not ansible_system is match("Linux")
+
+- name: "Wait for port 8000 to become open"
+  wait_for:
+    port: 8000
```

Comment:
8000 isn't open on Splunk Universal Forwarder because there's no web UI. If we want to make this portable, we should switch to 8089.